indico/queries/usermetrics.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from indico.client.request import (
+    GraphQLRequest,
+    PagedRequest,
+)
+from indico.types import BaseType
+from indico.types.user_metrics import UserSummary, UserSnapshot
+
+
+class _PagedUserSnapshots(BaseType):
+    """
+    Class to hold paged snapshot data to make parsing easier.
+    """
+    results: List[UserSnapshot]
+
+
+class _UserMetricsFilters(BaseType):
+    """Filters for fetching user metric data."""
+    date: str
+    user_id: str
+    user_email: str
+
+    def __init__(self, **kwargs):
+        self.date = kwargs.get('date', None).strftime('%Y-%m-%d') if kwargs.get('date', None) is not None else ""
+        self.user_id = kwargs.get('user_id', "")
+        self.user_email = kwargs.get('user_email', "")
+
+
+class GetUserSummary(GraphQLRequest):
+    """
+    Requests summary level information per-date of users in the app.
+    Includes enabled/disabled user counts, names of roles,
+    and number of users assigned to that role.
+
+    Args:
+        date (datetime): specific day to summarize.
+
+    """
+    query = """
+query GetUserSummary($date: Date){
+  userSummary(date: $date){
+    users{
+      enabled
+      disabled
+    }
+    appRoles {
+      role
+      count
+    }
+  }
+}
+    """
+
+    def __init__(self, **kwargs):
+        date = kwargs.get('date', None)
```

Comment:
why not explicitly call out `date` in the kwargs?
```suggestion
    def __init__(self, date=None):
```